src/backend/functions/change-requests.ts

```diff
@@ -49,7 +49,7 @@ const routes: ApiRoute[] = [
 const handler: Handler = async (event, context) => {
   try {
     return routeMatcher(routes, event, context);
-  } catch (error) {
+  } catch (error: any) {
```

Comment:
I believe there's an `Error` type

Reply:
![Screenshot (979)](https://user-images.githubusercontent.com/80832201/135371201-913bf376-441c-4a88-877d-0beeb00c6422.png)
If I try that then this happens. It seems like it has to be either 'any' or 'unknown'.

Reply:
@jamescd18 https://stackoverflow.com/questions/42618089/how-do-you-use-typed-errors-in-async-catch
TLDR; catch clause variables may not have a type annotation

Reply:
Thank you for clarifying! Let's go with just `any` then?